test/test.jl

```diff
@@ -32,13 +32,13 @@ a[1,1,1,1,1] = 10
 
 # Test printing of Pass results
 # Pass - constant
-@test contains(sprint(show, @test true), "Expression: true")
+#@test contains(sprint(show, @test true), "Expression: true")
```

Comment:
I deliberately removed the information about tests that pass because it was massively exceeding the rss limit for workers. That's why these tests are commented out.